plugins/XmlMaterialProfile/XmlMaterialProfile.py

```diff
@@ -648,6 +645,163 @@ def deserialize(self, serialized):
         for container_to_add in containers_to_add:
             ContainerRegistry.getInstance().addContainer(container_to_add)
 
+    @classmethod
+    def deserializeMetadata(cls, serialized: str, container_id: str) -> List[Dict[str, Any]]:
+        result_metadata = [] #All the metadata that we found except the base (because the base is returned).
+
+        #Update the serialized data to the latest version.
+        serialized = cls._updateSerialized(serialized)
+
+        base_metadata = {
+            "type": "material",
+            "status": "unknown", #TODO: Add material verification.
+            "container_type": XmlMaterialProfile,
+            "id": container_id
+        }
+
+        try:
+            data = ET.fromstring(serialized)
+        except:
+            Logger.logException("e", "An exception occurred while parsing the material profile")
```

Comment:
Change log, add "metadata" somewhere.

Reply:
The log already says that the error occurs in `deserializeMetadata`. The reason why this fails is an XML parsing error, so it doesn't matter whether it is in this one or the other.